docs/streams.html

```diff
@@ -398,6 +401,85 @@
+            </ul>
+        </li>
+        <li> Parameter updates in <code>StreamsConfig</code>:
+            <ul>
+                <li> parameter "zookeeper.connect" got deprecated </li>
+                <li> added many new parameters for metrics, security, and client configurations </li>
+            </ul>
+        </li>
+        <li> Changes in <code>StreamsMetics</code> interface:
+            <ul>
+                <li> removed methods: <code>#addLatencySensor()</code> </li>
+                <li> added methods: <code>#addLatencyAndThroughputSensor()</code>, <code>#addThroughputSensor()</code>, <code>#recordThroughput()</code>,
+                     <code>#addSensor()</code>, <code>#removeSensor()</code> </li>
+            </ul>
+        </li>
+        <li> New methods in <code>TopologyBuilder</code>:
+            <ul>
+                <li> added overloads for <code>#addSource()</code> that allow to specify stream specific <code>"auto.offset.reset"</code> policy </li>
+                <li> added methods <code>#addGlobalStore()</code> to add global <code>StateStore</code>s </li>
+            </ul>
+        </li>
+        <li> New methods in <code>KStreamBuilder</code>:
+            <ul>
+                <li> added overloads for <code>#stream()</code> and <code>#table()</code> that allow to specify stream specific <code>"auto.offset.reset"</code> policy </li>
+                <li> <code>#table()</code> always require to specify a store name </li>
+                <li> added method <code>#globalKTable()</code> to create a <code>GlobalKTable</code> </li>
+            </ul>
+        </li>
+        <li> New joins for <code>KStream</code>:
+            <ul>
+                <li> added overloads for <code>#join()</code> to join with <code>KTable</code> </li>
+                <li> added overloads for <code>#join()</code> and <code>leftJoin()</code> to join with <code>GlobalKTable</code> </li>
+            </ul>
+        </li>
+        <li> Aligned <code>null</code>-key handling for <code>KTable</code> joins:
+            <ul>
+                <li> like all other KTable operations, <code>KTable-KTable</code> joins do not throw an exception on <code>null</code> key records anymore, but drop those records silently </li>
+            </ul>
+        </li>
+        <li> New window type <em>Session Windows</em>:
+            <ul>
+                <li> added class <code>SessionWindows</code> to specify session windows </li>
+                <li> added overlaods for <code>KGroupedStream</code> methods <code>#count()</code>, <code>#reduce()</code>, and <code>#aggregate()</code>
+                     to allow session window aggregations </li>
+            </ul>
+        </li>
+        <li> Changes to <code>TimestampExtractor</code>:
+            <ul>
+                <li> method <code>#extract()</code> got second parameter </li>
+                <li> new default timestamp extractor class <code>FailOnInvalidTimestamp</code> </li>
```

Comment:
Well. It was one of the first thing after 0.10.1 release -- don't ask me about improved join semantics that get release with 0.10.2... That to merge the day after code freeze for 0.10.1 :)